### ISSUE TITLE
Issue building Pillow on Mac

### ISSUE THREAD

**Issue by aowen87** [visit-dav/issues-test#2135]
I went to run the test suite on Mac and it was not able to import PIL. I tried "import PIL" using the Python that build_visit had made for me and still no luck. I blew away my python build and had build_visit do it over. I noticed that it was not able to build Pillow (I guess we switched to Pillow). I tried building Pillow manually following the steps in the build_pillow function from bv_python.sh. I could not get the build to work because the Python that build_visit created for me was missing the ez_build stuff that lets "python setup.py build" work.

I was able to fix it like this:
1. Download https://bootstrap.pypa.io/ez_setup.py
2. python ez_setup.py

Then build pillow
3. python ./setup.py build
4. python ./setup.py install

The bottom line here is that the build_python may need to first make sure that ez_setup.py has been installed or the pillow build can fail.
https://pypi.python.org/pypi/setuptools

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Brad Whitlock
Original creation: 03/20/2015 03:56 pm
Original update: 04/01/2015 03:02 pm
Ticket number: 2186
